README.md

```diff
@@ -78,20 +78,23 @@ For more usage details, see our detailed [documentation](#documentation).
 
 ## Benchmarks
```

Comment:
I'd recommend viewing either the rich diff or the end result rendered as markdown.